Simplenote/LoginWindowController.m

```diff
@@ -7,11 +7,91 @@
 //
 
 #import "LoginWindowController.h"
+#import "SPConstants.h"
 #import "SPTracker.h"
 
+static CGFloat const SPLoginAdditionalHeight        = 40.0f;
+static CGFloat const SPLoginWPButtonWidth           = 270.0f;
+static NSString *SPAuthSessionKey                   = @"SPAuthSessionKey";
 
 @implementation LoginWindowController
 
+- (instancetype)init {
+    self = [super init];
+    
+    // Sanity check for accessing the root view
+    if (self.window.contentView.subviews.count < 1) {
```

Comment:
Q! can we move this to `windowDidLoad` or it's breaking there?

Reply:
> Q! can we move this to windowDidLoad or it's breaking there?

I tried overriding `windowDidLoad` and it didn't seem to ever get called? 🤷‍♂️

Reply:
I've digged a bit into this, and apparently, you need to use one of the designated initializers to get that method called (windowDidLoad).

For the record: because of the way this class is initialized, neither `awakeFromNib`, `windowWillLoad` / `windowDidLoad` nor `loadWindow` will get called.

Probably the only improvement we could do is move the Window Customization code to a method (and shrink a little bit the init), but the win is really marginal.

Squirrel still stands Dan, and thanks for checking!!! :shipit: